Vagrantfile

```diff
@@ -24,15 +25,21 @@ CONFIG = File.join(File.dirname(__FILE__), "config.rb")
 
 # Defaults for config options defined in CONFIG
 $num_instances = 1
+$num_big_instances = 0
 $instance_name_prefix = "core"
 $enable_serial_logging = false
 $share_home = false
 $vm_gui = false
 $vm_memory = 1024
 $vm_cpus = 1
+$vm_big_memory = 8192
+$vm_big_cpus = 2
 $vb_cpuexecutioncap = 100
 $shared_folders = {}
 $forwarded_ports = {}
+$update_channel = "alpha"
+$node_disks = 0
```

Comment:
Is this the number of disks (if so why is it defaulting to 0)?
If not then maybe it should be called `node_additional_disks` or something like that.

Reply:
Yes, it is the number of additional disks beyond the boot disk, so the name could be clearer.  How does `num_data_disks` and `data_disk_size` sound for names?  The `num_` prefix is already used elsewhere, I'm trying to blend in with the existing names.

Reply:
from the cluster i brought up earlier this morning from master, I already have 3 10GB disks.  how was that working before?
```
core@core-01 ~ $ lsblk -o name,type,size,partlabel
NAME    TYPE   SIZE PARTLABEL
sdd     disk    10G 
sdb     disk    10G 
sdc     disk    10G 
sda     disk  18.5G 
|-sda4  part     1G USR-B
| `-usr crypt 1016M 
|-sda2  part     2M BIOS-BOOT
|-sda9  part  16.1G ROOT
|-sda7  part    64M OEM-CONFIG
|-sda3  part     1G USR-A
|-sda1  part   128M EFI-SYSTEM
`-sda6  part   128M OEM
```

Reply:
The castle script always creates a `config.rb` configuration file that overrides these defaults, based on the `castle.default.config` and `castle.config` files.  These defaults only apply if you run `vagrant up` directly in the coreos-vagrant directory without creating a `config.rb` file.

Reply:
Yeah @dab-q, that is probably fine but maybe have a comment that gives a more textual explanation?